packages/plugins/rubygem-foreman_webhooks/rubygem-foreman_webhooks.spec

```diff
@@ -34,6 +34,9 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.15.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
```

Comment:
This upper boundary is preventing the build as the current version of it is 10.1.0. This probably needs to be lifted in the gems package.json as the dependencies are created from there.